src/view/items/element/specials/Input.js

```diff
@@ -5,4 +5,16 @@ export default class Input extends Element {
 		super.render( target, occupants );
 		this.node.defaultValue = this.node.value;
 	}
+	compare ( value, attrValue ) {
```

Comment:
There's a bit of whitespace mismatch from the style guide here. It always takes me a bit to switch into Ractive style mode :smile: